build/repositories.bzl

```diff
@@ -41,26 +41,15 @@ def github_cli_repositories():
             build_file_content = _SRCS_BUILD_FILE_CONTENT,
         )
 
-def _copyright_header(ctx):
```

Comment:
let's keep this block, but just remove the invocation (Line 99) so ce to ee merge will be simpler